px-data-grid.html

```diff
@@ -1268,6 +1268,12 @@
               }
             });
           }
+
+          setTimeout(() => {
```

Comment:
This works for Chome and FF, but not with Safari and IE.  Any ideas why?  If you set the timeout to something long like 1000 or 2000 it will work so there is some timing issue here.

Reply:
Fixed in https://github.com/vaadin/px-data-grid/pull/222/commits/9d012829e6fe2ad9d7b8acdb841107d33bdf95fe